packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -37,10 +37,47 @@ const propTypes = {
    * Index within the Menu Stack.
    */
   index: PropTypes.number.isRequired,
+  /**
+   * Bounding container for the menu, will use window if no value provided.
+   */
+  boundingRef: PropTypes.func,
+  /**
+   * Indicates if the menu content should set default focus on itself.
+   */
+  isFocused: PropTypes.bool,
+  /**
+   * Indicates if menu's height has been constrained by bounding container.
+   */
+  isHeightBounded: PropTypes.bool,
```

Comment:
Maybe I'm missing where this is determined, but I don't think Menu ever uses the isHeightBounded and isWidthBounded to the MenuContent component?

Reply:
These props are added by the Popup.